packages/components/style.css

```diff
@@ -93,3 +93,8 @@
     }
   }
 }
+
+/* wrong with neue montreal */
+._subpixel-antialiased {
+  -webkit-font-smoothing: unset;
```

Comment:
```suggestion
  -webkit-font-smoothing: unset;
  -moz-osx-font-smoothing: unset;
```

Reply:
Just checked, you're right, autoprefixer won't handle a `-webkit` prefixed property.